### DISCUSSION TITLE
Unable to find component

### DISCUSSION THREAD

**Discussion by chuckgriffith2** [livewire/livewire#4861] [1. Help]
in my view file when I use @livewire('perfs-edits’) or @livewire(‘livewire.perfs-edits’) I get 

Unable to find component: [perfs-edits] at
-------
if (! isset($_instance)) {
    $html = \Livewire\Livewire::mount('perfs-edits')->html();
} elseif ($_instance->childHasBeenRendered('kcfjJuQ')) {
--------
if I do @include(‘livewire.perfs-edits’) it succeeds so there is no spelling mistake.
why isn't it saying where it is looking for the view?

The installer puts the Component in app/App/Http/Livewire which seems odd but it hasn’t complained about that yet.
The demos online show it in app/Http/Livewire
The installer puts the view in app/resources/views/livewire
I tried putting the view under app/App/resources/views/livewire but that didn’t help.

The livewire config file says:
'class_namespace' => 'App\\Http\\Livewire’,
'view_path' => resource_path('views/livewire’),

view_path is then
/Users/chuckg/Sites/Submissions/resources/views/livewire

iMac % ls -la /Users/chuckg/Sites/Submissions/resources/views/livewire
total 16
drwxr-xr-x   3 chuckg  staff    96 Apr  5 08:12 .
drwxr-xr-x@ 16 chuckg  staff   512 Apr  4 12:46 ..
-rw-r--r--   1 chuckg  staff  6824 Apr  4 20:35 perfs-edits.blade.php

@livewireStyles loads properly
@livewireScripts calls for 
    <script src="/livewire/livewire.js?id=940557fc56b15ccb9a2d" data-turbo-eval="false" data-turbolinks-eval="false"></script>

There is a livewire/livewire/dist/livewire.js but no livewire/livewire.js
Besides the fact that the path is invalid, the file would have to be somewhere in the public folder for Apache to serve it and there is no file there named livewire anything.

I don't understand how this thing can be so broken and have no method of determining what the problem is.

**Comment by joshhanley** [comment]
@chuckgriffith2 something doesn't look right there, the component should be saved in `app/Http/Livewire` but you are saying yours is saving in `app/App/Http/Livewire`.

Can you share exactly what you typed in when you ran the `livewire:make` command?

Thanks!
